lib/rubygems/gemcutter_utilities.rb

```diff
@@ -267,6 +269,27 @@ def get_scope_params(scope)
     scope_params
   end
 
+  def get_mfa_params(email, password)
+    mfa_level = get_user_mfa_level(email, password)
+    params = {}
+    if mfa_level == "ui_only" || mfa_level == "ui_and_gem_sign"
+      selected = ask "Would you like to enable MFA for this key? [y/N]"
+      params["mfa"] = true if selected =~ /^[yY](es)?$/
+    elsif mfa_level == "ui_and_api"
+      params["mfa"] = true
```

Comment:
> If the user has an mfa level of `ui_and_api`, turns mfa on for the key without asking

I forgot to update the UI PR - the behaviour on the UI was changed so that we don't explicitly enable MFA if a user has `ui_and_api` ([discussion](https://github.com/rubygems/rubygems.org/pull/2846#discussion_r757738763))

Reply:
Ah gotcha! Updated